### PR TITLE
Add Netperf Testing

### PR DESCRIPTION
**Pull request by wizhaoredhat**
- Also removed the requirement to pass in the evaluator config

### CODE REVIEW

File: netperf.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+        if self.reverse:
+            logger.info(f"Reverse is not supported by Netperf")
+
+        logger.info(f"Running {self.cmd}")
+
+        self.exec_thread = ReturnValueThread(target=client, args=(self, self.cmd))
+        self.exec_thread.start()
+
+    def generate_output(self, data: str) -> IperfOutput:
```
Comment (SalDaniele) [inline netperf.py:83]: IperfOutput -> PerfOutput

File: netperf.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+    def output(self, out: common.TftAggregateOutput) -> None:
+        # Return machine-readable output to top level
+        assert isinstance(
+            self._output, IperfOutput
+        ), f"Expected variable to be of type IperfOutput, got {type(self._output)} instead."
+        out.flow_test = self._output
+
+        # Print summary to console logs
+        logger.info(f"Results of {self.ts.get_test_str()}:")
```
Comment (SalDaniele) [inline netperf.py:125]: Maybe it would be helpful for debugging to print a summary of the output here as we do for iperf tests.

**Review by SalDaniele — COMMENTED** [review]
LGTM

File: netperf.py (hunk 1 above)
Comment (wizhaoredhat) [reply netperf.py:84]: We will need to refactor this a bit to handle different result outputs from iperf and netperf. We can do this later.

**Pull request merged**